packages/framework/presence/src/core-interfaces/jsonDeserialized.ts

```diff
@@ -0,0 +1,115 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { InternalUtilityTypes } from "./exposedUtilityTypes.js";
+
+/**
+ * Options for {@link JsonDeserialized}.
+ *
+ * @alpha
+ */
+export interface JsonDeserializedOptions {
+	/**
+	 * Exact types that are managed by custom deserialization logic (beyond
+	 * {@link https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse|JSON.parse}
+	 * without a reviver). Only exact types matching specification will be
+	 * preserved unaltered.
+	 *
+	 * The default value is `never`.
```

Comment:
Nit
```suggestion
	 * @defaultValue `never`.
```

Reply:
I'm not sure that actually applies here. This type doesn't have a default, when this options type is passed to `JsonDeserialized`, then the "default" applies. That is also the only value use case for this type. I don't know the best way to express that. If `@defaultValue `never` is it cool. We'll want to make that change in #21725.